src/main/kotlin/com/cjbooms/fabrikt/generators/client/OkHttpSimpleClientGenerator.kt

```diff
@@ -230,6 +230,6 @@ data class SimpleClientOperationStatement(
                 requestBody.getPrimaryContentMediaType()?.key,
                 "toMediaType".toClassName("okhttp3.MediaType.Companion")
             )
-        } ?: this.add("\n.%N(ByteArray(0).%T())", verb, toRequestBody)
+        } ?: if (verb == "delete") this.add("\n.delete()") else this.add("\n.%N(ByteArray(0).%T())", verb, toRequestBody)
```

Comment:
This change seems out of place.  I suggest checking to see if a `delete` verb has any `Body` parameters in `addRequestStatement`.